pkg/pkg.bzl

```diff
@@ -344,9 +346,11 @@ pkg_deb_impl = rule(
         "depends_file": attr.label(allow_single_file = True),
         "suggests": attr.string_list(default = []),
         "enhances": attr.string_list(default = []),
+        "breaks": attr.string_list(),
```

Comment:
Can you make it default=[] for consistency.  Not that any of them need the default, but it will be easier for someone to understand in the future if they are all the same